mmpose/core/visualization/effects.py

```diff
@@ -3,6 +3,45 @@
 import numpy as np
 
 
+def screen_matting(img, color_low=None, color_high=None, color=None):
```

Comment:
What about moving these codes into tools/webcam/. For example in the node.py or utils.py.

Reply:
Done.